gce_ovf_import_tests/test_suites/ovf_import_test_suite.go

```diff
@@ -184,6 +184,25 @@ func TestSuite(
 			expectedMachineType:   "n1-standard-4",
 			expectedStartupOutput: "All tests passed!",
 		},
+		{
+			importParams: &ovfimportparams.OVFImportParams{
```

Comment:
@zoran15 an irrelative topic to this PR: can we call the ovf import cli tool instead of lib in gce_ovf_import_tests? That is closer to the real user experience. I am not sure whether current implementation has its specific purpose.

Reply:
I think it would be a good idea to refactor these tests to use gcloud using latest import wrapper. That would be the closest to what customers can expect.